tests/unit_tests/test_manual.py

```diff
@@ -20,7 +20,7 @@ def test_russian():
     model = yttm.BPE.train(TRAIN_DATA_PATH, MODEL_PATH, 50)
     tokenized_text = model.encode([test_text], output_type=yttm.OutputType.SUBWORD)
     expected_result = [
-        ["▁с", "обранный", "▁с", "об", "ран", "и", "е", "▁", "п", "р", "и", "бор"]
+        ["▁", "        ", "с", "об", "ранный", " ", "с", "об", "ран", "и", "е", " ", "п", "р", "и", "бор", "▁", "        "]
```

Comment:
Please make sure this result is expected (also in other tests)

Reply:
Yes, lgtm